samples/nrf9160/gnss/src/main.c

```diff
@@ -346,28 +348,32 @@ static void print_fix_data(struct nrf_modem_gnss_pvt_data_frame *pvt_data)
 					       pvt_data->datetime.seconds);
```

Comment:
In my PR you had suggested including speed accuracy, and using %04u for the year.

Reply:
I had already started modifying the sample output, but then decided that it's better to do a separate PR for that. I'm going to add some more information from the PVT at the same time, so let's keep that separate.